tests/sos_tests.py

```diff
@@ -51,27 +51,27 @@ def wrapper(self, *args, **kwargs):
     return decorator
 
 
-# pylint: disable=unused-argument
 def redhat_only(tst):
-    def wrapper(func):
+    def wrapper(*args, **kwargs):
         if _distro.name not in RH_DIST:
             raise TestSkipError('Not running on a Red Hat distro')
+        tst(*args, *kwargs)
```

Comment:
Aren't the tests run even without this explicit call of the method? (same applies to all three `*_only`).

Reply:
No:
```python
def my_decorator(func):
    print("my_decorator() called")

    def wrapper(*args, **kwargs):
        print("wrapper() called")

    return wrapper


@my_decorator
def my_func():
    print("my_func() called")


my_func()
```
```
$ python3.10 decorators.py
my_decorator() called
wrapper() called
```

Reply:
I think what was meant was the change of `*args, **kwargs` changes in the functions, do we need this change?

Reply:
Yes; if this is used to annotate a class method:
```python
class MyTests(...):
    @ubuntu_only
    def test_smth(self):
        # ...
```
the `self` parameter will be missed & the tests will fail. I included `kwargs` for completeness, I don't think any functions currently take kwargs.

Reply:
yeah, no arguments are being passed anywhere in the sos code for any of these decorators, hence probably the question of the change here

Reply:
If I got it right, the change is not needed "per se" or for the sake of this PR, but it allows potential usage of arguments in decorators, right?

Could be..